### PR TITLE
fix relative path to examples

### PR DESCRIPTION
**Pull request by yeesian**
see #28

### CODE REVIEW

File: test/testexamples.jl [hunk 1]
```diff
@@ -1,5 +1,5 @@
-for test_file in readdir("../examples/")
+for test_file in readdir(Pkg.dir("KNITRO", "examples"))
```
Comment (mlubin) [inline test/testexamples.jl]: Use ``@__FILE__`` instead of ``Pkg.dir("KNITRO")``.
E.g., ``joinpath(dirname(@__FILE__), "..", "examples")``

Comment (yeesian) [inline test/testexamples.jl]: Interesting, didn't know you can do that. I was following https://github.com/JuliaStats/DataFrames.jl/blob/9b2cc92679b5dc2433c87f83e15b469e0797dda0/benchmark/io.jl#L5

Comment (mlubin) [inline test/testexamples.jl]: Well yes, but that's bad style. The package might not be in the same place as ``Pkg.dir`` says.

**Comment by yeesian** [comment]
squash and rebase?

**Comment by mlubin** [comment]
Sure

**Pull request merged**